<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <!--
    <remap from="/plan/steering" to="/steering"/>
    <remap from="/plan/speed" to="/speed"/>
    -->

    <node name="urc_center_keeper" pkg="rr_iarrc" type="urc_center_keeper" output="screen">
        <param name="speed" type="double" value="1.0"/>
        <param name="PID_kP" type="double" value="0.05"/>
        <param name="left_angle_offset" type="double" value="-0.6"/>
        <param name="right_angle_offset" type="double" value="-0.6"/>
        <param name="camera_left_subscription" type="string" value="/camera_left/image_color_rect"/>
        <param name="camera_right_subscription" type="string" value="/camera_right/image_color_rect"/>
    </node>
</launch>
